src/mail/view/MailFoldersView.ts

```diff
@@ -128,6 +131,27 @@ export class MailFoldersView implements Component<MailFolderViewAttrs> {
 		return result
 	}
 
+	private renderAddButton(attrs: MailFolderViewAttrs): Child | null {
+		return m(
+			".folder-row.flex.flex-row.mlr-button.border-radius-small.state-bg.button-height.click",
```

Comment:
why don't we use `IconButton`?

we need to either use `button` as a tag or add `aria-role`

Reply:
Makes sense! We should comment on why we then and fix the semantics